src/main/java/org/jvnet/hudson/test/LoggerRule.java

```diff
@@ -284,4 +297,22 @@ public void describeTo(org.hamcrest.Description description) {
         }
     }
 
+    /**
+     * Delegates to the given Handler but filter out records higher or equal to its initial level
```

Comment:
Did you see #145 and specifically verify this behavior?

Reply:
I didn't see it before, but afaics all the FINE logs were ok when I verified it.

Reply:
Great, just verifying that you tested more carefully than I did in #138. :disappointed: